### DISCUSSION TITLE
Play Two Playlists at same time

### DISCUSSION THREAD

**Discussion by DJMic** [AzuraCast/AzuraCast#7118] [Q&A]
Hi,
I've 2 playlists (playlist-1 and playlist-2).
I would like to play in this order at specific time (ex: every day at 10:30 AM).
Playlist-1 contains one song and Playlist-2 contains many songs but play one.
How can I do this ?
Sometimes, Azuracast plays playlist-2, playlist-1. And sometimes, it's good order (playlist-1, playlist-2).
How can I resolve this without use Advanced Playlists ?

**Comment by RM-FM** [comment]
I don't think you can achieve this without Advanced playlists / Liquidsoap scripting. The only thing you could try is to force Playlist-1 to be played immediately by enabling "Interrupt other songs to play at scheduled time.". Schedule Playlist-2 for the same time but without forcing it to play immediately. Would that work?

**Reply by unudelaradio** [reply]
It would be great if **"Interrupt other songs to play at scheduled time"** feature would act as a _skip_. But (unfortunately) once the song from Playlist 1 is done playing, the paused song will resume from where it was interrupted. Only after the interrupted song resumes, the song from Playlist 2 will be played. 

What i usually do, to avoid wrong order is to set _Weight 25_ for Playlist 1 and _Weight 1_ for Playlist 2. I'm doing this when i create the playlists: I save the playlist as General Rotation to set the weight, and then change the Playlist Type to Once per Hour (at the specified minute). 

Also (don't know if this has any effect and also sounds funny) i keep Playlist 2 disabled till Playlist 1 is loaded for the first time in Queue and once i see it loaded i enable very fast Playlist 2 to be sure this will be the first time order and somehow i hope AutoDJ remembers this order next time. :))))

**Reply by DJMic** [reply]
Thanks for your answer. I'm going to try this method. But I don't think Weights interfere good order

**Comment by DJMic** [comment]
The solution above does not work. I don't think Weights interfere good order. My Playlist-1 contains some jingles. Playlist-2 contains some specific songs. I would like to schedule to play one item from PL-1 and one item from PL-2 in this specific order at 10:30 AM. Is it possible ? Sometimes, PL-2 starts before PL-1.
